recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.10.0" %}
+
+package:
+    name: pycairo
+    version: {{ version }}
+
+source:
+    git_url: git://git.cairographics.org/git/pycairo
+
+build:
+    number: 0
+    skip: true                  # [win or py2k]
+    script:
+        - python setup.py install
```

Comment:
Please add the arguments `--single-version-externally-managed --record=record.txt`.